tests/bsim/bluetooth/host/adv/encrypted/css_sample_data/testcase.yaml

```diff
@@ -0,0 +1,10 @@
+tests:
+  bluetooth.host.adv.encrypted.css_sample_data:
+    build_only: true
+    tags:
+      - bluetooth
+    platform_allow:
+      - nrf52_bsim
```

Comment:
@aescolar should this also be?

```suggestion
      - nrf52_bsim/native
```

Reply:
It does not matter as it is the only option.